WalletCore/Classes/ViewModel/CashOut/CashOutBankAccountViewModel.swift

```diff
@@ -52,6 +58,31 @@ public class CashOutBankAccountViewModel {
                 
                 return true
         }
+        
+        isValidIBAN = iban.asObservable()
+            .map { iban in
+                iban.isValidIban()
+        }
+        
+        isValidBIC = bic.asObservable()
+            .map { bic in
+                bic.isValidBic()
+        }
+        
+        isValid = Observable.combineLatest([
```

Comment:
You can use the closure after ```Observable.combineLatest``` to produce the single Boolean:
```isValid = Observable.combineLatest(isValidFields, isValidIBAN, isValidBIC) { $0 && $1 && $2 }```, or you can use ```.zip``` if you are sure, that all observables will emit altogether :)

Reply:
p.s. notice, I don't use an array in the ```.combineLatest``` method.